docs/README.md

```diff
@@ -258,6 +169,35 @@ cancel | string | https://example.org/51/cancel | Called on cancelled payment
 
 See [an example payload and response](/examples?id=create)
 
+#### Redirect and callback URL parameters
+
+Once the payment is complete, or cancelled, normally the client browser is redirected to the merchant provided URL. If merchant has provided a callback URL, it will be called too.
```

Comment:
normally the client browser is redirected to -> the client browser is normally redirected to